rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -154,6 +154,10 @@
         "request_cache": {
           "type" : "boolean",
           "description" : "Specify if request cache should be used for this request or not, defaults to index level setting"
+        },
+        "search_after": {
+          "type" : "list",
+          "description" : "A comma-separated list of sort values that indicates where the sort of the top hits should start"
```

Comment:
Actually, as a query string param, it'd have to be a comma-separated list of sort values.  Honestly, I'm not sure we can reliably pass these values in via query string parameters, eg it is possible for a sort value to be `null`, but that would become the string `"null"` as a query string param.